tests/models/conftest.py

```diff
@@ -0,0 +1,48 @@
+"""Fixtures for the model tests"""
+import gzip
+import io
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture(name="content")
+def fixture_content() -> str:
+    """Return some content for a file"""
+    _content = (
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt"
+        " ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ull"
+        "amco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehende"
+        "rit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaec"
+        "at cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+    )
+    return _content
+
+
+@pytest.fixture(name="file_path")
+def fixture_file_path(project_dir: Path) -> Path:
+    """Return the path to a non existing file"""
+    return project_dir / "a_file.txt"
+
+
+@pytest.fixture(name="gzipped_file_path")
+def fixture_gzipped_file_path(file_path: Path) -> Path:
```

Comment:
Same here: I would expect to get a path to an existing file

Reply:
Fixed